genome/gene_statistic.py

```
@@ -0,0 +1,455 @@
+            "I": ARSC(C=4, N=0, S=0),
+            "P": ARSC(C=3, N=0, S=0),
+            "F": ARSC(C=7, N=0, S=0),
+            "M": ARSC(C=3, N=0, S=1),
+            "W": ARSC(C=9, N=1, S=0),
+            "G": ARSC(C=0, N=0, S=0),
+            "C": ARSC(C=1, N=0, S=1),
+            "U": ARSC(C=1, N=0, S=0),
+            "J": ARSC(C=4, N=0, S=0),
+            "B": ARSC(C=2, N=0.5, S=0),
+            "Z": ARSC(C=3, N=0.5, S=0),
+            "U": ARSC(C=1, N=0, S=0),
```

Ruff (0.11.9)

Dictionary key literal `"U"` repeated

Remove repeated key literal `"U"`

(F601)